_sass/components/_footnotes.scss

```diff
@@ -58,4 +53,5 @@ body {
 .footnote {
   vertical-align: super; // 2
   font-size: smaller;
+  @extend .link-charlie;
```

Comment:
Expected item on line 56 to appear before line 54. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets